From: Robert Luberda <robert@debian.org>
Date: Mon, 21 Mar 2011 10:27:27 +0100
Subject: 0029 Generate hex in fix8bit

Make fix8bit -7 generate hexadecimal sequences instead of octal ones.
---
 languages/fix8bit.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/languages/fix8bit.c b/languages/fix8bit.c
index 6090c5b..da149df 100644
--- a/languages/fix8bit.c
+++ b/languages/fix8bit.c
@@ -182,7 +182,7 @@ static void to7bit ()		/* Convert from 8-bit sequences */
 	{
 	ch &= 0xFF;
 	if (ch >= 0x80)
-	    (void) printf ("\\%3.3o", (unsigned) ch);
+	    (void) printf ("\\x%2.2X", (unsigned) ch);
 	else
 	    (void) putchar (ch);
 	}
